src/app.py

```diff
@@ -26,6 +27,7 @@ def create_app(env_name):
   app.register_blueprint(campaign_blueprint, url_prefix='/api/v1/campaigns')
   app.register_blueprint(request_blueprint, url_prefix='/api/v1/requests')
   app.register_blueprint(price_converter_blueprint, url_prefix='/api/v1/price_converter')
+  app.register_blueprint(contributor_blueprint, url_prefix='/api/v1/contributor')
```

Comment:
This section is getting kinda big. Possible refactor in the future? Maybe we can place the blueprint imports and blueprint app registration in its own file?